bigquery/storage/managedwriter/integration_test.go

```diff
@@ -709,8 +709,9 @@ func testInstrumentation(ctx context.Context, t *testing.T, mwClient *Client, bq
 		if err != nil {
 			t.Errorf("view %q RetrieveData: %v", tv.Name, err)
 		}
-		if len(metricData) > 1 {
-			t.Errorf("%q: only expected 1 row, got %d", tv.Name, len(metricData))
+		if mlen := len(metricData); mlen != 1 {
+			t.Errorf("%q: expected 1 row of metrics, got %d", tv.Name, mlen)
+			continue
```

Comment:
I was gonna say to change to ` > 0` ( not empty ) because is easier to read, but as we strictly want to check for it having exactly one that looks good. Also the main fix is not the bound check, but adding the `continue` clause. Talking out load as I was trying to understand the PR. 

LGTM.